src/test/mochitest/head.js

```diff
@@ -363,6 +363,24 @@ async function waitForPaused(dbg) {
   );
 }
 
+/*
+ * useful for when you want to see what is happening
+ * e.g await waitForever()
+ */
+function waitForever() {
```

Comment:
this seems like a method used while debugging, but not something that is used in the code...

Reply:
they both are

Reply:
they both are, i think they are helpful patterns though that we should make easier for newcomers